docs/source/installation_instructions.rst

```diff
@@ -0,0 +1,91 @@
+.. The purpose of this file is to provide instructions on how to install the software necessary to access and run
+   the pytentiostat software package.
+
+Installation Instructions
+=========================
+
+.. seealso::
+   Instructions are listed below for installing Miniconda, but the full user documentation for Anaconda is available at
+   https://conda.io/projects/conda/en/latest/user-guide/install/index.html
+
+System Requirments
+------------------
+
+Before installing Anaconda, the following system requirements must be met:
+
+* 32- or 64-bit computer
+* 400 MB disk space
+* Windows, MacOS, or Linux
+* Python 2.7, 3.4, 3.5, 3.6, or 3.7
+* pycosat
+* PyYaml
+* Requests
+
+If you do not already have one of the required versions of Python, follow the instructions below.
+
+.. note::
+
+   We used Python 3.7 for the Pytentiostat software, so it is encouraged to download that version to ensure
+   compatibility.
+
+Python Installation
+--------------------
+
+Go to the following page that corresponds with the operating system that you are working on.
+
+* Windows:      https://www.python.org/downloads/windows/
+* MacOS:        https://www.python.org/downloads/mac-osx/
+* Linux/UNIX:   https://www.python.org/downloads/source/
+
+Click on the download link for the latest release of Python 3.7. Double click the .exe file and follow prompts to proceed
+with installation.
+
+Install pycosat, PyYaml, and Requests.
+---------------------------------------
+
+In the start menu, find and open the python prompt.
+
+Miniconda Installation
+-----------------------
+Once all system requirements are met, navigate to the following conda installation page corresponding to the operating
+system you are using.
+
+* Windows:  https://conda.io/projects/conda/en/latest/user-guide/install/windows.html
+* MacOS:    https://conda.io/projects/conda/en/latest/user-guide/install/macos.html
+* Linux:    https://conda.io/projects/conda/en/latest/user-guide/install/linux.html
+
+Following the instructions on the documentation page, click the installer link for Miniconda. Then click on the
+link for the 32- or 64- bit installer. This downloads an executable file. Once the file is downloaded, you should verify
+the installer hashes using the instructions found at https://conda.io/projects/conda/en/latest/user-guide/install/download.html#hash-verification.
+
+Once the installer hashes have been verified, double click the .exe file and follow on-screen prompts to complete the
+installation. If you do not have settings preferences, continue with defaults.
+
+Verify the Installation was Successful
+---------------------------------------
+
+When the installation is complete, you can test whether it was successful by navigating to the anaconda prompt in the
```

Comment:
I don't necessarily recommend using the anaconda terminal, though we could argue about this.  It is more general though to give instructions using either a bash style terminal for mac/linux or a windows terminal for windows users.

Reply:
We can certainly do that. I just figured since the anaconda terminal would already be installed, it would be simple for everyone to use. It also ran the code smoothly without crashing the code by closing the plot window.

Reply:
I don't think we want Anaconda as a dependency though.